catalog/plugin/catalog-plugin-metacard-validation/src/main/java/ddf/catalog/metacard/validation/MetacardValidityMarkerPlugin.java

```diff
@@ -70,8 +71,10 @@ public CreateRequest process(CreateRequest input)
                 } catch (ValidationException e) {
                     // If validator is not explicitly turned on by admin, set invalid and allow through
                     if (checkEnforcedMetacardValidators(metacardValidator)) {
-                        Boolean validationErrorsExist   = e.getErrors()   != null && !e.getErrors().isEmpty();
-                        Boolean validationWarningsExist = e.getWarnings() != null && !e.getWarnings().isEmpty();
+                        Boolean validationErrorsExist =
```

Comment:
Why use the Boolean object here? It looks like it is just causing autoboxing twice. If you switched this over to the primitive, it'd be faster.